docs/content/topics/scripting/dependencies.md

```diff
@@ -9,102 +9,123 @@ aliases:
+  the worker's node process and dependencies.
 
-- as the Brigade worker is capable of modifying the state of the cluster, be mindful 
-of any external dependencies added through `brigade.json`
+- Dependencies are dynamically installed on every Brigade script execution -
+  this means if the dependencies added are large, and the event frequency is
+  high for a particular project, it might make sense to make a pre-built Docker
+  image that already contains the dependencies. See the [Workers] document for
+  further details on how to do so.
 
-- for now, the only part of `brigade.json` used is the `dependencies` object - it means the rest of the file 
-can be used to pass additional information to `brigade.js`, such as environment data - however, as the `brigade.json` 
-file is passed in the source control repository, information passed there will be available to anyone with access to the repository.
-
-- all dependencies added in `brigade.json` are dynamically installed on every Brigade build - this means if the dependencies added 
-are large, and the build frequency is high for a particular project, it might make sense to make a pre-built Docker image that 
-already contains the dependencies, [as described in this document](workers.md).
+[package.json]: https://docs.npmjs.com/cli/v7/configuring-npm/package-json
+[Workers]: /topics/scripting/workers
 
 ## Using local dependencies from the project repository
 
-Local dependencies are resolved using standard Node [module resolution](https://nodejs.org/api/modules.html#modules_all_together),
-with one change: the worker's `node_modules` directory is added as a fallback, so `brigade.js`—and any local dependencies—can resolve modules installed via `brigade.json`.
-This approach works great for using dependencies that are not intended to be external packages, and which are located in the project repository. 
+Local dependencies are resolved using standard Node [module resolution], with
+one change: the worker's `node_modules` directory is added as a fallback, so
+`brigade.js` - and any local dependencies - can resolve modules installed via
+`package.json`.
+
+This approach works great for using dependencies that are not intended to be
+external packages, and which are located in the project repository. 
 
-Let's consider the following scenario: we have a JavaScript file located in `/local-deps/circle.js`, where `local-deps` is a directory at the root of our git repository. In our `brigade.js` file, we can use any exported method or variable from that package by simply using a `require` statement, just like in any other JavaScript project.
+Let's consider the following scenario: we have a JavaScript file located in
+`/local-deps/circle.js`, where `local-deps` is a directory at the root of our
+git repository. In our Brigade script, we can use any exported method or
+variable from that package by simply using a `require` statement, just like in
+any other JavaScript project.
```

Comment:
The example might be more straightforward if the filed were located in the `.brigade/` directory beside the `brigade.js/ts`.

Your example works, but I think that keeping Brigade scripts, config, etc. "separate" from the rest of the project (i.e. tucked away in their own directory) leads to a tidier project layout. I think we should encourage that, even if only implicitly via demonstartion.

Reply:
That's fair.  I can certainly revise per your suggestion.  However, I can imagine scenarios where the Brigade project uses a git repo with local deps/js assets that may also be used by other software/CI tools/etc. associated with the repo.  In that case, I'd assume they'd want the local deps to be standalone and not tightly coupled to Brigade (i.e. not in the .brigade dir).  WDYT?